ocrd/ocrd/cli/workspace.py

```diff
@@ -159,6 +167,54 @@ def workspace_add_file(ctx, file_grp, file_id, mimetype, page_id, force, local_f
     )
     workspace.save_mets()
 
+# ----------------------------------------------------------------------
+# ocrd workspace add-group
+# ----------------------------------------------------------------------
+
+@workspace_cli.command('add-group')
+@click.option('-m', '--mimetype', help="Media type of the file", required=True)
+@click.option('-f', '--force', help="Continue even if mets:file same ID already exists", default=False, is_flag=True)
+@click.argument('file_grp_dir', type=click.Path(dir_okay=True, file_okay=False, readable=True, resolve_path=True), required=True)
+@pass_workspace
+def workspace_add_file_group(ctx, mimetype, force, file_grp_dir):
+    """
+    Add a directory of files as a fileGrp to METS.
+
+    Convention:
+        FILEGRP_DIR must reside within the workspacee
+        fileGrp ID = last folder name component
+        page ID = filename sans extension
+        file ID = sanitized concatenation of fileGrp ID and page ID
+    """
+    workspace = Workspace(ctx.resolver, directory=ctx.directory, mets_basename=ctx.mets_basename, automatic_backup=ctx.automatic_backup)
+
+    file_grp_dir = Path(file_grp_dir).resolve()
+
+    ws_dir = Path(ctx.directory).resolve()
+    print(file_grp_dir)
+
+    # XXX will raise ValueError if file_grp_dir isn't relative to ctx.directory
+    file_grp_dir.relative_to(ws_dir)
+    file_grp = clean_id(file_grp_dir.name)
```

Comment:
maybe validate that fileGrp does not exist yet?

Reply:
`--overwrite` parameter will remove file groups before adding files. For performance reasons, no checks on the actual METS currently implemented. But `--force` will be passsed on to `OcrdMets.add_file`.